backend-db/modules/db/couchdb/design/index.js

```diff
@@ -171,7 +171,7 @@ const designDocs = {
                 view: {
                     map: function (doc) {
                         if (doc.$doctype === 'team') {
-                            emit(doc.name, doc);
+                            emit(doc._id, doc);
```

Comment:
We have the following code in /backend-db/modules/db/couchdb/index.js :
```
getTeam: id => adapter.getOneEntityById('Team', '_design/byTeamId/_view/view', id),
getAllTeams: () => adapter.getEntities('Team', '_design/all/_view/view', {sorted: true}),
```
You will break sorting by name with that changes "emit(doc._id, doc);"

I've already fixed this in https://github.com/letsdoitworld/World-Cleanup-Day/pull/275/files

Reply:
I suggest revert that line and use my fix.